docs/source/model_design/vivarium_overview/vivarium_versus_other_model_types/index.rst

```diff
@@ -64,31 +74,114 @@ Over the duration of a given microsimulation run, simulant attributes are dynami
 Simulation components are interdependent, and relationships between attributes are also based on best available real-world 
 evidence. For example, disease progression is a function of simulant characteristics and treatment; survival depends on stage of disease and other characteristics. 
 
+As the simulation proceeds, various simulant attributes change and these changes are recorded through observers (to read more about observers, visit [insert link here]).
+
 .. note::
 
   Are you feeling confused by some of the vocabulary on this page? Please visit our `Vivarium glossary <https://vivarium-research.readthedocs.io/en/latest/glossary/index.html>`_ 
   for some definitions of microsimulation vocabulary!
 
-What are the advantages and disadvantages of individual-based microsimulations?
--------------------------------------------------------------------------------
+What are the advantages and disadvantages of microsimulations?
+--------------------------------------------------------------
+
+Microsimulation, described above, is a special kind of model that falls under the broader umbrella of **individual-based modeling (IBM)**.
+
+In this section, we will elaborate on what differentiates microsimulation from other forms of IBM, such as agent-based simulation. 
```

Comment:
As I read it, most of the pros/cons in this section are comparing microsimulation vs other types of statistical models such as compartmental models or multiplication models. I'm not sure I understand exactly the pros and cons of the way we do things vs ABM, but I think ABM can do most of these "advantages."